Deeploy/Targets/Snitch/TileConstraints/ReshapeTileConstraint.py

```
@@ -0,0 +1,143 @@
+class ReshapeTileConstraint(TileConstraint):
+    """Tile constraint for Reshape operation - a NOP that just reinterprets data layout."""
+
+    @staticmethod
+    def addGeometricalConstraint(tilerModel: TilerModel, parseDict: Dict, ctxt: NetworkContext) -> TilerModel:
+
+        inputBufferName = parseDict['data_in']
+        outputBufferName = parseDict['data_out']
+
+        pointer: List[str] = []
+
+        for key, value in parseDict.items():
```

Ruff (0.14.14)

[warning] Loop control variable `key` not used within loop body

Rename unused `key` to `_key`

(B007)